DirectProgramming/DPC++FPGA/Tutorials/Features/scheduler_target_fmax/sample.json

```diff
@@ -0,0 +1,55 @@
+{
+  "guid": "1ADD31DD-8D83-45E8-8AC4-6972C80354EB",
```

Comment:
Please confirm you generated a unique GUID and didn't just copy/paste from another design

Reply:
Yes I generated a unique GUID in sample.json.